__tests__/app.test.js

```diff
@@ -38,61 +36,86 @@ describe("/api/topics", () => {
-        })
-      })
-})
+describe("Error Handling", () => {
+  test("ERROR: 400 when given an invalid pathway", () => {
+    return request(app)
+      .get("/api/not-news-address")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
+});
 
-describe('/api/articles/:article_id', ()=>{
-  test('GET: 200 responds with the selected article when given a valid id', ()=>{
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 responds with the selected article when given a valid id", () => {
     return request(app)
-    .get('/api/articles/3')
-    .expect(200)
-    .then(({body})=>{
-      expect(body.article.article_id).toBe(3)
-      expect(body.article).toMatchObject({
-        title: "Eight pug gifs that remind me of mitch",
-        topic: "mitch",
-        author: "icellusedkars",
-        body: "some gifs",
-       created_at: expect.any(String),
-       votes: expect.any(Number),
-        article_img_url:
-          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
-      })
-    })
-  })
-  test('ERROR: 404 responds with an error when a non-existent id is given', ()=>{
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.article_id).toBe(3);
+        expect(body.article).toMatchObject({
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("ERROR: 404 responds with an error when a non-existent id is given", () => {
     return request(app)
-    .get('/api/articles/89')
-    .expect(404)
-    .then(({body})=>{
-      expect(body.msg).toBe('no article with this ID')
-    })
-  })
-  test('ERROR: 400 responds with an error when given an invalid id', ()=>{
+      .get("/api/articles/89")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("no article with this ID");
+      });
+  });
+  test("ERROR: 400 responds with an error when given an invalid id", () => {
+    return request(app)
+      .get("/api/articles/numbers-only")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("/api/articles", () => {
+  test("GET: 200 retrieves all the articles", () => {
     return request(app)
-    .get('/api/articles/numbers-only')
-    .expect(400)
-    .then(({body})=>{
-      expect(body.msg).toBe('Bad Request')
-    })
-  })
-})
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+       const {articles} = response.body
+        expect(articles).toHaveLength(13);
```

Comment:
Great addition of a length check so you don't get a false positive with an empty array